tests/api/test_payment.py

```diff
@@ -459,19 +464,121 @@ def test_query_payments(payment_dummy, permission_manage_orders, staff_api_clien
     assert payment_ids == [payment_id]
 
 
-@patch("saleor.graphql.payment.resolvers.gateway_get_client_token")
-def test_query_payment_client_token(mock_get_client_token, user_api_client):
+@pytest.fixture
```

Comment:
Shouldn't be at the beginning of the file?

Reply:
Is it used by test above?